src/Row.js

```diff
@@ -46,7 +46,14 @@ export default class Row extends Component {
   _panResponder = PanResponder.create({
     onStartShouldSetPanResponder: () => !this._isDisabled(),
 
-    onMoveShouldSetPanResponder: () => !this._isDisabled(),
+    onMoveShouldSetPanResponder: (e, gestureState) => {
+      if (this._isDisabled()) return false;
+
+      const vy = Math.abs(gestureState.vy)
+      const vx = Math.abs(gestureState.vx)
+
+      return this._active && (this.props.horizontal ? vx > vy : vy > vx);
+    },
```

Comment:
> This should allow for child horizontal pan responders (ie swiping).

Do you tested? Or it is just a guess?

Reply:
Yes I was able to use react-native-swipe-list-view now.

Reply:
Hey @bradbumbalough , would you mind posting a minimal example? I have been struggling to produce a working sortable and swipeable list for some time now. It would be very appreciated.

Reply:
Tested with react-native-swipe-list-view too, works, cherry-picked to master. Thanks @bradbumbalough

Reply:
Hi, I tried it in order to make react-native-sortable-list and react-native-swipe-list-view (with SwiperRow), the row swipes well but is not sortable anymore. Is there a working example somewhere ? Thanks !